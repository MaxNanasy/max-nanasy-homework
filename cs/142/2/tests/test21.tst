main() {	
var i, j: int;	
if (i < = j) 
{		j = i;	}
}