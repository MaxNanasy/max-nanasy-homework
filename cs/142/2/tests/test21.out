program
   declarations
      newType
         type
   statementSequence
      statement
         ifStatement
            condition
               expression
                  term
                     factor
                        designator
Error: =
